ABProcessCore.js

```diff
@@ -170,6 +170,12 @@ module.exports = class ABProcessCore extends ABMLClass {
          fn = () => {
             return true;
          };
+
+      // If parent, merge connections
+      if (this.process && this.key === "SubProcess") {
```

Comment:
this is the list of other process tasks, by merging the "connections" of the parent process, subprocess gains access to those tasks... and therefore can use that data when inserting child records.


I am concerned that this could cause performance issues, not sure how to test it though